boa/src/builtins/bigint/mod.rs

```diff
@@ -31,6 +31,7 @@ impl BuiltIn for BigInt {
 
     fn attribute() -> Attribute {
         Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE
+        // maybe false
```

Comment:
What did you mean by this?

Reply:
Sorry, I was debugging a test and I forgot to remove this comment haha